src/components/textarea/styles.ts

```diff
@@ -0,0 +1,79 @@
+import { css } from '@linaria/core'
+import { styled } from '@linaria/react'
+
+export const elTextAreaHasError = css``
+
+export interface TextAreaCSSProperties {
+  '--textarea-max-rows': number
+  '--textarea-min-rows': number
+}
+
+export const ElTextArea = styled.textarea`
+  // NOTE: These are public CSS variables that we use to allow CSS-only consumers
+  // to define the maximum and minimum rows the text area should resize between.
+  // We do this ourselves instead of leveraging Linaria's dynamic styles because
+  // that approach results in randomly-named CSS variables, which would not provide
+  // a friendly interface for CSS-only consumers.
+  --textarea-max-rows: infinity;
+  --textarea-min-rows: 2;
+
```

Comment:
**note:** The code comment below is another important decision. While I strongly believe there should be _zero_ developer-defined, component-specific design tokens in Elements, I don't think that should prevent us from leveraging CSS variables in a "private" manner to aide the definition of our styles.

In order to try and communicate they are "private"/"implementation detail" CSS variables, I've prefixed them with the standard double underscore, `__`. Not sure if that's something we want to do or not.

If we'd prefer not to do this at all, I can remove these and just use the real design tokens directly in all the places they're referenced.

Reply:
It does look a bit odd syntactically but the comment explains fine. I would suggest as long as they are outside the root scope it's probably fine.

Reply:
@willmcvay I agree the syntax looks weird. We could choose to use `private` as a prefix? i.e. `--private-textarea-border-width` 🤔